src/routes/players/[userId]/+page.svelte

```diff
@@ -1,3 +1,4 @@
+<!-- Shot CSS file-->
```

Comment:
Remove

Reply:
Have it there so I can figure out which one is which